web-chat/src/custom.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@livechat/ui-kit';
```

Comment:
To keep it consistent with js-waku, name the file `types.d.ts` and add it to a `types` folder.

You probably also need to update the `web-chat/tsconfig.json` file with

```
 "typeRoots": ["node_modules/@types", "src/types"]
```

like `./tsconfig.json`.

Reply:
Ok should be done now, sorry :)